tests/cmdline/params/options/test_verbosity.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Tests for the `-v/--verbosity` option."""
+
+from click.testing import CliRunner
```

Comment:
would be good to use the `tests.cmdline.commands.conftest.run_cli_command` fixture instead. You can check `tests.cmdline.commands.test_plugin.py` for an example.

Reply:
added in 08ab9da8a017a03372f54d4e8b642a48941e5bad